Documentation/Pages/InstallationGuide.rst

```diff
@@ -19,7 +19,20 @@ Target group: **Developers**
 Installation
 ------------
 
-1. Install the extension from TER (sources are available on `our GitHub repository <https://github.com/AOEpeople/TYPO3_Restler>`_.
+1. Install the extension from TER (sources are available on `our GitHub repository <https://github.com/AOEpeople/TYPO3_Restler>`_. In the context of a fully Composer setup, add in your root composer.json the following lines before a ```composer update```.
+
+::
+
+	"require": {
+		...
+		"aoe/restler": "dev-master",
+		"luracast/restler": "3.0.0-RC6",
+	}
```

Comment:
Merged this documentation amendment with the following changes:

- instead of requiring "dev-master", I have changed to "1.*"
- removed the requirement for luracast/restler in the root composer.json (it is already required by aoe/restler).

Any comments?